### PR TITLE
Add redirect to New Feature Onboarding checklist

### PR DESCRIPTION
**Pull request by zachmargolis**

### CODE REVIEW

File: _articles/new-feature-onboarding.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+title: "New Feature Onboarding"
+description: "Checklist for new features, specifically focused on documentation needed for security review"
+layout: article
+category: "Product"
```
Comment (aduth) [inline _articles/new-feature-onboarding.md:5]: I would _love_ to have something like this for general product features, but this document seems very tailored to security reviews. Maybe it should be categorized accordingly?

**Review by aduth — APPROVED** [review]

File: _articles/new-feature-onboarding.md (hunk 1 above)
Comment (zachmargolis) [reply _articles/new-feature-onboarding.md:5]: Are you suggesting this?
```suggestion
category: "Security"
```

Comment (aduth) [reply _articles/new-feature-onboarding.md:5]: Yes 👍

Comment (aduth) [reply _articles/new-feature-onboarding.md:5]: I think there's a lot of overlap with a hypothetical product feature checklist though, so I don't feel _too_ strongly about it 🤷

Comment (zachmargolis) [reply _articles/new-feature-onboarding.md]: When that checklist exist, we can update + recombine!

**Pull request merged**